docs/source/documentation/platform/infrastructure/run-data-product-in-platform.html.md.erb

```diff
@@ -1,48 +1,65 @@
 review_in: 6 months
 weight: 15
 ---
 
 # <%= current_page.data.title %>
 
-## Context
-We want a compute process that is scalable and event driven to run the data product. The data product will be sent to Data Platform for processing. After processing, the data should be available in a suitable format accessible to data consumers. The processing will include receiving raw data, applying transformations, cleansing and finally storing data in a correct location and format.
+## Data as a Product: Vision
 
-<strong>The architecture will be built in the `data-platform-development`, a sandbox account within MoJ's Modernisation Platform, which is deployed using the [modernisation-platform-environments](https://github.com/ministryofjustice/modernisation-platform-environments) repository</strong>
+Each Data Product will require a [specification](https://github.com/ministryofjustice/data-platform-products/blob/main/data-products/_docs/product-specification.md) (still under refinement) alongside [other defined pieces of descriptive metadata captured at registration](https://github.com/ministryofjustice/data-platform-products/tree/main/data-products#defining-a-data-product).
 
-## Data Producer Configuration Flow
+Data Products:
 
-![Data platform general diagram](../images/data-platform-configuration-flow.png "Data platform configuration Flow")
+* May be multi-component, e.g. consisting of multiple tables
+* Can consist of structured or unstructured data (or a combination)
+* Are controlled and updated by Data Owners, Data Stewards, and Data Custodians.
+* Include data metadata
 
-**For now** - Data product configuration will be sent to the Data Platform S3 bucket via [GitHub Actions](https://github.com/ministryofjustice/data-platform-products/blob/main/.github/workflows/data-product-example-2-poc.yaml).
+We want to design the Data Product management process around a user-facing API.
+This API will allow users to register, update, and retrieve information about a Data Product.
 
-The data product configuration (Zip file) will be stored in the following structure:
-> `${DATA_PLATFORM_PRODUCTS}/code/${DATA_PLATFORM_PRODUCT_NAME}/${ZIPFILE}.zip`
+The data sent to this API as part of registering or updating a data product will flow through standard Data Lake zones: Landing -> Raw -> Curated.
 
-Event Bridge connection to bucket should be enabled. Once the file lands in the landing zone, S3 sends an event to Amazon Event Bridge whenever data events happen in the bucket. We will use Event Bridge rules to route events Object Created `PutObject` events to Lambda.
+> Implementation: The architecture will be built in the `data-platform-development` aws account, a sandbox account within [MoJ's Modernisation Platform](https://user-guide.modernisation-platform.service.justice.gov.uk/#modernisation-platform), which is deployed within the [modernisation-platform-environments](https://github.com/ministryofjustice/modernisation-platform-environments) repository
```

Comment:
Let's not be environment-specific here (sandbox). Suggest something like:

The architecture will be built in the Data Platform's [Modernisation Platform](https://user-guide.modernisation-platform.service.justice.gov.uk/#modernisation-platform) [environment(s)](https://github.com/ministryofjustice/modernisation-platform-environments/tree/main/terraform/environments/data-platform)